docs/index.rst

```diff
@@ -8,14 +8,16 @@ hosting RPM family content types.
 Features
 --------
 
-* :ref:`sync-publish-workflow` with "RPM Content" including RPMs and Errata
+* :ref:`sync-publish-workflow` with "RPM Content" including RPMs, Advisories, Modularity, and Comps
 * :ref:`Versioned Repositories <versioned-repo-created>` so every operation is a restorable snapshot
 * :ref:`Download content on-demand <create-remote>` when requested by clients to reduce disk space.
 * Upload local RPM content :ref:`easily <one-shot-upload-workflow>`
 * Copy and organize RPM content into various repositories
```

Comment:
we don't have copy yet, what about re-phrasing to 'Add, remove and organize...'

Reply:
I can, but as I did not write that and it is outside of this issue, should it be done in another PR?

Reply:
I think this is in the scope of this PR. Tanya probably overlooked when writing the description of the task, in any case it is the assignee duty to thoroughly check whether the feature list is up to date with the reality.